torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -672,17 +672,11 @@ def rewrite_node(node):
                     return g.call_function(ttnn.squeeze, args=(args[0], args[1]))
 
             if node.target == torch.ops.aten.unsqueeze.default:
-                if args[0].op == "get_attr":
-                    value = getattr(gm, args[0].target)
-                    input_size = value.size()
-                else:
-                    input_size = args[0].meta["val"].size()
-
-                output_size = node.meta["val"].size()
-                output_size = list(output_size)
-                if output_size[-1] == input_size[-1]:
-                    return g.call_function(ttnn.reshape, args=(args[0], output_size))
-                return None
+                output_shape_num_element = node.meta["val"].numel()
+                if output_shape_num_element == 0:
```

Comment:
Can you please open a PR into metal with this check in the Op itself? 
Good to leave this check here for now too, but lets move to tt-metal

Reply:
Ok, I'll add this check at tt-metal

Reply:
I open in there
https://github.com/tenstorrent/tt-metal/pull/15289